std/internal/math/biguintcore.d

```diff
@@ -1456,14 +1455,12 @@ void mulInternal(BigDigit[] result, const(BigDigit)[] x, const(BigDigit)[] y)
             addAssignSimple(result[done .. done + y.length + chunksize], partial);
             done += chunksize;
         }
-        delete scratchbuff;
```

Comment:
Here (and below) I see both path delete `scratchbuff`, so merge the two lines in one. Again, use `malloc`/`free`.